datasets/multi_nli/README.md

```diff
@@ -52,75 +74,82 @@ basis for the shared task of the RepEval 2017 Workshop at EMNLP in Copenhagen.
 
 ### Languages
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+The dataset contains samples in English only.
 
 ## Dataset Structure
 
-We show detailed information for up to 5 configurations of the dataset.
-
 ### Data Instances
 
-#### plain_text
-
 - **Size of downloaded dataset files:** 216.34 MB
 - **Size of the generated dataset:** 73.39 MB
 - **Total amount of disk used:** 289.74 MB
 
-An example of 'validation_matched' looks as follows.
+Example of a data instance:
+
 ```
 {
-    "hypothesis": "flammable",
-    "label": 0,
-    "premise": "inflammable"
+    "promptID": "31193",
+    "pairID": "31193n",
+    "sentence1": "Conceptually cream skimming has two basic dimensions - product and geography.",
```

Comment:
Can you keep the hypothesis/premise/label field names please ?
This is a convention we use for NLI datasets